### ISSUE TITLE
Abstract should appear **after** author affiliation in docx

### ISSUE THREAD

**Issue by VEZY** [pandoc-scholar/pandoc-scholar#42]
The abstract appears before the authors affiliation in docx documents, but it should appear **after**. I tried to do it and spent hours understanding how AST and lua filters work without finding a good solution. 
I saw that authors affiliations and correspondence are positioned in the `body` using the `author-info_blocks.lua` filter, but the abstract is in `meta`. Would it be possible to keep affiliations and correspondence in `meta` so the abstract appears after ? 

I tried to put the abstract back into the body, but then the template doe not recognize it as an abstract so it is wrongly formatted.

One similar solution is to **not** use the `abstract-to-meta.lua` filter and to put the abstract in the markdown file. But of course it yields the same output as the solution I tried before. 

Unfortunately the abstract is most of the time formatted differently than the text from paragraphs in the templates from editors, so it is not a viable solution to put it in the body (unless we can format it the proper way). 

docx is an important format for me to share the paper to colleagues that just want to use word as always before. 

Could you take a look at that ?

**Comment by jcolomb** [comment]
There seem to be no template for word document, is that the problem?

**Comment by VEZY** [comment]
Yes indeed, but I tried with one and I couldn't figure out how to place the abstract after the authors affiliation. There seems to be no options to place the authors affiliations in docx templates when looking at the Pandoc documentation for [`--reference-doc`](https://pandoc.org/MANUAL.html#option--reference-doc). So unless working with filters, I'm not sure it is doable.

**Comment by tarleb** [comment]
Indeed, the lack of "proper" templating for docx output is a problem.

I guess we could try to fix this particular problem by emitting raw OOXML to wrap the abstract (i.e., replicate pandoc's behavior in a way that's suitable for our use-case).

**Comment by jcolomb** [comment]
Indeed, the https://github.com/benmarwick/rrtools project did not adress this either.
